src/DynamoCoreWpf/UI/Themes/Modern/InPorts.xaml

```diff
@@ -198,6 +198,12 @@
 
 
             <!--  A border, to initiate the node autocomplete command -->
+            <Rectangle x:Name="NodeAutoCompleteHover"
```

Comment:
Can we perhaps get away without creating this rectangle ? And rely only on MainBorderHighlightOverlay and NodeAutoCompleteMarker border elements ?

Reply:
The border highlight doesn't extend over the edge enough, so it flickers on and off. I can nest a shape within the border maybe? 

The connection border extends beyond the node, but it is only hitTestVisible after click

Reply:
Do you think I can nest the elements within a grid instead @BogdanZavu ? Then the grid can serve as the element that triggers visibility of the inner border element?